bamboo_twig_i18n/src/TwigExtension/I18n.php

```diff
@@ -85,4 +87,33 @@ public function formatDate(TwigEnvironment $env, $date, $type = 'medium', $forma
     return NULL;
   }
 
+  /**
+   * Gets a translation of the entity.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity for which to get the translation.
+   * @param string|null $langcode
+   *   (optional) The language code of the translation to get. NULL (default) means to use
+   *   the user interface language for the page.
```

Comment:
`user interface language` or `current language`. Are they the same thing?

Reply:
it's the same.